py3status/modules/coin_market.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=degraded {name}] ${price_usd:.2f} {percent_change_24h}')
+    format_separator: show separator only if more than one (default ' ')
+    markets: comma separated list of supported markets https://coinmarketcap.com
+        (default ['btc'])
+    request_timeout: time to wait for a response, in seconds (default 5)
+
+Format placeholder:
+    {format_coin} format for cryptocurrency coins
+
+format_coin placeholders:
+    {24h_volume_usd}      eg 1435150000.0
+    {available_supply}    eg 16404825.0
+    {id}                  eg bitcoin
+    {last_updated}        eg 1498135152
+    {market_cap_usd}      eg 44119956596.0
+    {name}                eg Bitcoin
+    {percent_change_1h}   eg -0.17%
+    {percent_change_24h}  eg -1.93%
+    {percent_change_7d}   eg +14.73%
+    {price_btc}           eg 1.0
+    {price_usd}           eg 2689.45
```

Comment:
Really, what happened... If you compared https://api.coinmarketcap.com/v1/ticker/?convert=gbp&limit=1 to the placeholder list ( https://api.coinmarketcap.com/v1/ticker/?limit=1 )... There are extra `_usd` placeholders to all converted currencies... `{price_usd}`, `{24h_volume_usd}` and `{market_cap_usd}`. I didn't notice that before.